generic_config_updater/gu_common.py

```diff
@@ -43,6 +43,7 @@ def __eq__(self, other):
 class ConfigWrapper:
     def __init__(self, yang_dir = YANG_DIR):
         self.yang_dir = YANG_DIR
+        self.sonic_yang_with_loaded_models = None
```

Comment:
>sonic_yang_with_loaded_models [](http://example.com/codeflow?start=13&length=29)

Is `sonic_yang_model` good enough?

Reply:
I don't think so. `sonic_yang_model` refers to the model but here we are referring to `SonicYang` object [code](https://github.com/Azure/sonic-buildimage/blob/3aec72879d19e956e531b907f3f372706c916072/src/sonic-yang-mgmt/sonic_yang.py#L13)

This variable will hold an instance of `sonic_yang` but with the models loaded hence the name